src/components/Navigator/NavigatorCard.vue

```diff
@@ -91,6 +91,14 @@
               @clear="clearFilters"
             />
           </div>
+          <button
```

Comment:
Interesting proposal, @dobromir-hristov 
Would you move the button responsibility to `Navigator.vue` component instead? Could you develop why?

`BaseNavigatorCard` is already a generic component in case we want to re-use the navigator card into a different part of the website.

Reply:
@dobromir-hristov I'm working on updating this branch/PR and have a similar question to Marina based on this feedback.

Where would a better alternative for this button be? Would you suggest making a slot in the navigator so that the `DocumentationTopic` view component could utilize it for this?

If we were to move this somewhere else, should the filter also move with it?

Reply:
I think the DocumentationTopic should be the one responsible for rendering it yes. It has all the data related to it already.

Reply:
Something like [this](https://github.com/mportiz08/swift-docc-render/commit/521abcf66c428104da042ad00583549055461d60) maybe? (Had a little trouble figuring out how to test the higher level behavior with all the slots and nesting)

Reply:
You can remove now all unnecessary computeds and props and so on.